### PR TITLE
added aсordion in farm table

### PR DESCRIPTION
**Pull request by malytinKonstantin**

### CODE REVIEW

File: src/components/farmingTable/FarmingTable.tsx [hunk 1]
```diff
@@ -86,25 +83,56 @@ export const FarmingTable: React.FC<IProps> = observer((props) => {
...
+          >
+            {asset.name}{' '}
+            {asset.earnFarm && <img className="flash" src={flash} alt="" />}
+          </div>
+          {/* <div className="active">{asset.earnFarm.toString()}</div> */}
+          <div
+            className="earned-rewards"
+            // TODO: implements it
+            // onKeyUp={() => getThisReward(summary.earnedRewards)}
+            // onClick={() => getThisReward(summary.earnedRewards)}
```
Comment (cryptopower80) [inline src/components/farmingTable/FarmingTable.tsx:103]: Would it be possible to include the author of the comment `TODO (username):`? so we can keep all tracked

File: src/components/farmingTable/FarmingTableStyles.jsx [hunk 2]
```diff
@@ -114,9 +188,10 @@ export const MainTableRow = styled.div`
     }
   }
 `
+// 0.5fr 0.5fr 0.5fr
```
Comment (cryptopower80) [inline src/components/farmingTable/FarmingTableStyles.jsx:191]: is this comment necessary?

File: src/components/farmingTable/FarmingTable.tsx [hunk 3]
```diff
@@ -86,25 +83,56 @@ export const FarmingTable: React.FC<IProps> = observer((props) => {
...
+          className={accordion.includes(asset.name) ? 'open' : ''}
+        >
+          <div
+            className="name"
+            title={asset.earnFarm ? 'Earn FARM: true' : undefined}
+          >
+            {asset.name}{' '}
+            {asset.earnFarm && <img className="flash" src={flash} alt="" />}
+          </div>
+          {/* <div className="active">{asset.earnFarm.toString()}</div> */}
```
Comment (cryptopower80) [inline src/components/farmingTable/FarmingTable.tsx:98]: same comment as above

File: src/components/farmingTable/FarmingTable.tsx (hunk 1 above)
Comment (belbix) [reply src/components/farmingTable/FarmingTable.tsx:103]: I guess author can be changed with time
will be cool to link a task

**Comment by belbix** [comment]
recreated https://github.com/harvestfi/dashboard/pull/159

**Pull request closed without merging**